### DISCUSSION TITLE
Business shared folders with identical names

### DISCUSSION THREAD

**Discussion by sbakker** [abraunegg/onedrive#2015] [Q&A] [answered]
I haven't run into this yet, but I have just configured onedrive and managed to sync both my own drive and a business shared folder. I noticed this shared folder showed up under my `~/OneDrive` folder, which I guess is fine, but it did raise questions:

 1. What happens if I have a folder `Foo` and someone shares their folder `Foo` with me? Can I then still sync that shared folder, or will this clash with my personal `Foo` folder?
 2. If two users share their own `Foo` folder with me, how can I sync both if they have the same name?

**Comment by abraunegg** [comment, marked as answer]
@sbakker 
> I noticed this shared folder showed up under my ~/OneDrive folder, which I guess is fine, but it did raise questions

Correct - however, where there is concern with this, you can also run multiple client configurations, so that you store your own OneDrive content in 'sync_dir=1' vs the OneDrive Business Folders being stored in 'sync_dir=2'

To assist with this, this is why there is the following feature request: https://github.com/abraunegg/onedrive/issues/1251

 > 1. What happens if I have a folder Foo and someone shares their folder Foo with me? Can I then still sync that shared folder, or will this clash with my personal Foo folder?

It will clash and request an item to be renamed somewhere .. you will need to decide what to change the root folder name to.

> 2. If two users share their own Foo folder with me, how can I sync both if they have the same name?

The only option would be to run multiple configurations.
